src/utils/storage.js

```diff
@@ -0,0 +1,47 @@
+function setItem(key, value) {
+  localStorage.setItem(key, JSON.stringify(value));
```

Comment:
did you use this file because you were dealing with `localStorage` ?

Reply:
Yes that's the reason